modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -532,6 +616,123 @@ SHOW USERS YIELD roles, user
 WHERE 'admin' IN roles
 RETURN user AS adminUser
 ----
+.Result
+[role="queryresult" options="header,footer", width="100%", cols="2m"]
+|===
+|adminUser
+|"neo4j"
+1+a|Rows: 1
+|===
+======
+
+[[access-control-list-user-auth-providers]]
+=== Listing User Auth Providers
+
+To inspect users' xref:authentication-authorization/auth-providers.adoc[Auth Providers], use `SHOW USERS WITH AUTH`. This command will produce a row per user per Auth Provider and will yield the following two columns in addition to those output by `SHOW USERS`:
+
+[options="header", width="100%", cols="2a,4,2m,^.^,^.^"]
+|===
+| Column
+| Description
+| Type
+| Community Edition
+| Enterprise Edition
+
+| provider
+| The name of the auth provider.
+| STRING
+| {check-mark}
+| {check-mark}
+
+| auth
+| A map containing configuration for the user. E.g. dn of the user for an `ldap` auth provider, the unique external identifier for an `oidc` auth provider, or password status for a native auth provider.
+
+| MAP
+| {check-mark}
+| {check-mark}
+|===
+
+.Show users with auth
+======
+[source, cypher, role=noplay]
+----
+SHOW USERS WITH AUTH
+----
+
+.Result
+[role="queryresult" options="header,footer", width="100%", cols="2m,3m,3m,2m,2m,3m,4m"]
+|===
+|user
+|roles
+|passwordChangeRequired
+|suspended
+|home
+|provider
+|auth
+|"neo4j"
+|["admin","PUBLIC"]
+|false
+|false
+|<null>
+|"native"
+|{
+"password": "***",
+"changeRequired": false
+}
+|"jack"
+|["PUBLIC"]
+|false
+|false
+|<null>
+|"native"
+|{
+"password": "***",
+"changeRequired": false
+}
+|"jack"
+|["PUBLIC"]
+|false
+|false
+|<null>
+|"oidc1"
+|{
+"id": "jacksIdForOidc1"
+}
+7+a|Rows: 3
+|===
+======
+
+.Show user with auth using filtering
+======
+Show all of the users with the `oidc` Auth Provider.
+[source,cypher,role=noplay]
+----
+SHOW USERS WITH AUTH
+WHERE provider = 'oidc1'
+----
+
+.Result
+[role="queryresult" options="header,footer", width="100%", cols="2m,3m,3m,2m,2m,3m,4m"]
+|===
+|user
+|roles
+|passwordChangeRequired
+|suspended
+|home
+|provider
+|auth
+|"jack"
+|["PUBLIC"]
+|false
+|false
+|<null>
+|"oidc1"
+|{
+"id": "jacksIdForOidc1"
+}
+7+a|Rows: 1
+|===
+
 ======
 
 [NOTE]
```

Comment:
I think this note should move up to the main section and not listing auth section?

Reply:
to me this note says _"thats all for listing users. If you actually wanted to list their privileges, then do `SHOW USER name PRIVILEGES` etc."_

and so I could't find a place for it at one of the higher levels. Any ideas? 

Or we could re-word it to be a bit more descriptive?

Reply:
since the with auth is a subsection this note now lies in the subsection so to me it gives the feel of it being part of the subsection and not the whole listing users in general (which is what it should be, it's not specific to the with auth part)?

Maybe just move it up to the syntax table (or expand what is there) and remove it from down here alltogether?

Reply:
when you mention _syntax table_, do you mean the `SHOW USER PRIVILEGES` syntax table on line 103? If so then the need for this note disappears? (there's nothing from the note to add to the `SHOW USER PRIVILEGES` syntax table which isn't already there, right?)

Reply:
yeah, that one. I was more thinking if we wanted to switch 
```
For more information, see xref:authentication-authorization/manage-privileges.adoc#access-control-list-privileges[Listing privileges].
``` 
to something closer to 
```
The `SHOW USER name PRIVILEGES` command is described in xref:authentication-authorization/manage-privileges.adoc#access-control-list-privileges[Listing privileges].
```
from the note here, and then remove the note here

to be more explicit about it not being covered here at all and only over there